app/controllers/ops_controller.rb

```diff
@@ -829,6 +829,7 @@ def identify_tl_or_perf_record
   end
 
   def get_session_data
+    # TODO, import tasks_options from generic session
```

Comment:
Given https://github.com/ManageIQ/manageiq-ui-classic/pull/470/files#r105725558 .. can you just remove `tasks_options` from ops completely please?